content/src/service/ServiceImpl.ts

```diff
@@ -14,15 +14,16 @@ import {
 import { AuthChain, Authenticator } from 'dcl-crypto'
 import { EnvironmentConfig } from '../Environment'
 import { runReportingQueryDurationMetric } from '../instrument'
+import { NOT_ACTIVE } from '../ports/activeEntities'
 import { bufferToStream, ContentItem } from '../ports/contentStorage/contentStorage'
 import { FailedDeployment, FailureReason } from '../ports/failedDeploymentsCache'
 import { Database } from '../repository/Database'
 import { DB_REQUEST_PRIORITY } from '../repository/RepositoryQueue'
 import { AppComponents } from '../types'
-import { CacheByType } from './caching/Cache'
 import { getDeployments } from './deployments/deployments'
 import { DeploymentOptions } from './deployments/types'
 import { EntityFactory } from './EntityFactory'
+import { DELTA_POINTER_RESULT, DeploymentResult as PointerChanges } from './pointers/PointerManager'
```

Comment:
Why is `DeploymentResult` imported as `PointerChanges`? Note that `PointerChanges` is already defined in `types.ts` and generates confusion